core/templates/services/validators.service.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('Validators service', () => {
     });
     vs = TestBed.get(ValidatorsService);
     /* eslint-disable dot-notation */
-    AppConstants['INVALID_NAME_CHARS'] = '#xyz';
+    AppConstants['INVALID_NAME_CHARS'] = ['#', 'x', 'y', 'z'];
```

Comment:
Since you're changing this anyway could you do:
```suggestion
    AppConstants.INVALID_NAME_CHARS = ['#', 'x', 'y', 'z'];
```

Reply:
done